demo/scripts/controls/contextMenu/ResetListPlugin.ts

```diff
@@ -22,17 +23,35 @@ export default class ResetListPlugin implements ContextMenuProvider<ContextMenuI
 
         if (safeInstanceOf(node, 'HTMLLIElement')) {
             const list = this.editor.getElementAtCursor('ol,ul', node);
-            if (safeInstanceOf(list, 'HTMLOListElement') && list.start > 1) {
-                items.push({
-                    key: 'resetList',
-                    name: 'Reset list number',
-                    onClick: () => {
-                        this.editor?.addUndoSnapshot(() => {
-                            this.editor.focus();
-                            list.start = 1;
-                        }, ChangeSource.Format);
+            if (safeInstanceOf(list, 'HTMLOListElement')) {
+                items.push(
+                    {
+                        key: 'resetList',
+                        name: 'Reset list number',
+                        onClick: () => {
+                            this.editor?.addUndoSnapshot(() => {
+                                this.editor.focus();
+                                setOrderedListNumbering(this.editor, list, 1);
```

Comment:
Since we are adding a new format API, this API should handle everything such as focus to editor, and add undo snapshot.

Reply:
Added the editor.focus and the addUndoSnapshot to the function in this commit.
https://github.com/microsoft/roosterjs/commit/d48eaee2a387d9f40c59a8e3e5b378d5a9941952